### DISCUSSION TITLE
Изменение цвета стрелочки мыши

### DISCUSSION THREAD

**Discussion by vindsa** [freefelt/YouType#30] [Ideas]
Добрый день. Подскажите, пожалуйста, а возможно ли добавить функционал, чтобы цвет самой стрелочки мыши менялся в зависимости от языка? Например, если раскладка EN - цвет всегда черный, если RU - всегда синий? Другими словами, чтобы язык не появлялся при наведении на поле, а всегда можно было его определить по цвету указателя. Почему возник вопрос: когда пользуешься горячими клавишами, тогда не приходится наводить стрелочку на поле, чтобы определить язык. Цвет указателя сможет помочь с этим

**Comment by freefelt** [comment]
Привет! Спасибо за предложение. Я гляну как будет возможность, но есть вероятность, что так не получится сделать

**Reply by vindsa** [reply]
Добрый день!) Подскажите, пожалуйста, есть ли апдейт по вопросу, реально ли сделать изменение цвета стрелки в зависимости от языка?
